docs/csharp/language-reference/keywords/override.md

```diff
@@ -12,7 +12,7 @@ ms.assetid: dd1907a8-acf8-46d3-80b9-c2ca4febada8
 ---
 # override (C# Reference)
 
-The `override` modifier is required to extend or modify the abstract or virtual implementation of an inherited method, property, indexer, or event.
+The `override` modifier is required to extend or modify the abstract or virtual implementation of an inherited method, property, indexer or event.
```

Comment:
@volkan-yazilim I'm afraid most of the removed commas (so called Oxford commas) are placed correctly and should stay. For more information, see the docs style guide:
https://docs.microsoft.com/en-us/style-guide/punctuation/commas#use-a-comma